content/response-to-reviewers.md

```diff
@@ -0,0 +1,74 @@
+In this revision we will clarify text to highlight this association and be distinct on the difference between machine learning methods that make the association and methods that are used to perform tasks.
+
+>**at least references 1, 2, 5,6 and 2 are concerned with methods for graphs not necessarily knowledge graphs, in which the edge label (i.e., relation is essential to its definition).
+
+We acknowledge that these references use general methods for knowledge graphs; however, we use these references to exemplify the fact that the graphs used are considered knowledge graphs.
+
+>**Figure 1 doesn't show the relationship direction, For example, "causes", "binds" and other relations don't clearly specify the source and destination nodes which can be confusing.
+>Ideally, a knowledge graph should show that. 
+>Please have a look at some knowledge graphs reviews in literature. 
+>For example, https://arxiv.org/pdf/1503.00759.pdf 
+
+Figure one uses the same network schema as used in Project Rephetio (https://elifesciences.org/articles/26726).
+The authors on that project intentionally kept the edges as undirected; however,
+we agree that some relationships can have ambiguous directionality (e.g. upregulates/downregulates can infer gene upregulates disease or disease upregulates a gene).
+For this revision, we plan to update the text to state that our example knowledge graph can have bidirectionality.
+
+>**"relatively precise data, but in low quantifies"?
+
+This statement means the quality of curated data is of high grade, but the amount of data inserted into databases is quite small. 
+In other words, manual curation provides results that have high precision but low recall.
+The quantifies is a typo and should be quantities and will be fixed when the revision processed is finished.
+
+>**what you refer to as "unifying techniques" is relational learning, I don't see why you refer to it in such an ambiguous way. 
+>Furthermore, grouping the techniques into three is so broad and doesn't correctly represent knowledge graphs methods. 
+>For example, matrix factorization and deep learning. 
+>Matrix factorizations such as isomap, PCA, SVD and others are not knowledge graph representation techniques, but dimensionality reduction techniques.
+> More importantly, you don't show how they can be applied or used in the context of knowledge graphs as claimed?
+
+We agree that the title for that section is a bit broad and we updated the text to accommodate the suggestion.
+You are correct in stating that these methods are not knowledge graph specific; however, they can be used on knowledge graphs to associate nodes to dense vectors (low dimensional space).
+We provided a reference that used SVD and other matrix factorization methods to perform this association for link prediction tasks and node classification tasks.
+In this revision we will update text to be more explicit on how these class of methods can be used on knowledge graphs.
```

Comment:
Put the text that you are revising here too so your reader doesn't have to go back and forth between the documents.